packages/office-ui-fabric-react/src/components/TextField/TextField.styles.tsx

```diff
@@ -185,16 +190,7 @@ export function getStyles(props: ITextFieldStyleProps): ITextFieldStyles {
         display: 'flex',
         flexDirection: 'row',
         alignItems: 'stretch',
-        position: 'relative',
-        selectors: {
-          ':hover': {
-            selectors: {
-              [HighContrastSelector]: {
-                borderColor: 'Highlight'
-              }
-            }
-          }
-        }
+        position: 'relative'
```

Comment:
Why was this removed?

Reply:
I moved it town to 206 inside of `!disabled` otherwise disabled controls get a hover border 
![textfield](https://user-images.githubusercontent.com/1434956/69282424-282d8980-0b9f-11ea-8abd-ee8baec5ebe8.gif)